.github/workflows/build.yml

```
@@ -404,7 +404,7 @@ jobs:
       - name: Check coverage threshold
         run: |
           pnpm nyc check-coverage -t ./coverage/nyc \
-            --branches 99.57 \
+            --branches 99.7 \
```

```suggestion
            --branches 99.6 \
```
sadly main reduced it again 😕

---

So do we leae it at this point and no further improvements needed?